src-ghc/Pact/Persist/SQLite.hs

```diff
@@ -105,8 +107,8 @@ decodeBlob :: (FromJSON v) => SType -> IO v
 decodeBlob (SText (Utf8 t)) = liftEither (return $ eitherDecodeStrict' t)
 decodeBlob v = throwDbError $ "Expected text blob, got: " <> viaShow v
 
-encodeBlob :: ToJSON a => a -> SType
-encodeBlob a = SText $ Utf8 $ BSL.toStrict $ encode a
+encodeBlob :: B.ByteString -> SType
```

Comment:
The name `encodeBlob` with type  `B.ByteString -> SType` seems like it would take any bytestring (including non-utf8) and convert it to an `SType` (presumably by base64 encoding it).

I think this function was confusingly named before, although the type signature before made it harder to accidentally misuse the function. WDYT?